config/freertos/nucleo_f767zi/package.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>firmware</name>
+  <version>0.0.0</version>
+  <description>Micro-ROS dependecies for FreeRTOS and STM32F767ZI Nucleo Board</description>
+  <maintainer email="pablogarrido@eprosima.com">Eden Desta</maintainer>
```

Comment:
I guess that we don't have the same mail.

Reply:
Sorry, I missed that one.